test/integration/sourcemap/snapshot.spec.ts

```diff
@@ -24,49 +26,62 @@ const cases: Array<[string, string]> = [
 
 describe("Source map snapshot tests", () => {
     for (const [fileName, sample] of cases) {
-        describe(fileName, () => {
```

Comment:
Also it explicitly state changes, that may affect BC (intentionnaly or not). There are not much efforts needed to maintain curent snapshot tests. We have very limited set of snapshots. It would cause maintenance complexity if we decide to keep more snapshots, than we currrently have.

Reply:
Ok, lets keep this.